nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/pom.xml

```diff
@@ -59,20 +58,24 @@
             <artifactId>nifi-record</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.apache.nifi</groupId>
-            <artifactId>nifi-hadoop-record-utils</artifactId>
-            <version>1.9.0-SNAPSHOT</version>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>18.0</version>
         </dependency>
+
+        <dependency>
```

Comment:
This dependency can be removed, since Kudu doesn't use any of the "Hadoop Resources" like the HDFS/Hive/HBase components, and you're only referencing UGI from hadoop-commons. The nifi-security-utils module has classes in the org.apache.nifi.security.krb package that can be used to log in with a KDC, perform PrivelegedExceptionActions via the KerberosAction class, and handle automatic relogins when tickets expire.  I have a PR up now that adds some usability modifications to KerberosAction.  You could base this PR off of my Kerberized DBCPConnectionPool branch (https://github.com/jtstorck/nifi/tree/NIFI-5985) to work with those changes.

Reply:
That looks reasonable. I updated the JIRA to indicate that it is blocked by NIFI-5985.